core/io/file_access_zip.cpp

```diff
@@ -38,7 +38,6 @@
 ZipArchive *ZipArchive::instance = nullptr;
 
 extern "C" {
```

Comment:
Like for `namespace {`, that's something where we might want to preserve the spacing for readability.